2021-12-02 23:33:34,478 INFO    MainThread:42917 [wandb_setup.py:_flush():71] setting env: {}
2021-12-02 23:33:34,478 INFO    MainThread:42917 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-02 23:33:34,478 INFO    MainThread:42917 [wandb_init.py:_log_setup():371] Logging user logs to /Users/shy/Library/Mobile Documents/com~apple~CloudDocs/my python programming projects/DTI_pro/wandb/run-20211202_233334-oo8z3ppe/logs/debug.log
2021-12-02 23:33:34,479 INFO    MainThread:42917 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/shy/Library/Mobile Documents/com~apple~CloudDocs/my python programming projects/DTI_pro/wandb/run-20211202_233334-oo8z3ppe/logs/debug-internal.log
2021-12-02 23:33:34,479 INFO    MainThread:42917 [wandb_init.py:init():404] calling init triggers
2021-12-02 23:33:34,479 INFO    MainThread:42917 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {'max_epochs': 150, 'batch_size': 100}
2021-12-02 23:33:34,479 INFO    MainThread:42917 [wandb_init.py:init():449] starting backend
2021-12-02 23:33:34,479 INFO    MainThread:42917 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-02 23:33:34,493 INFO    MainThread:42917 [backend.py:ensure_launched():199] starting backend process...
2021-12-02 23:33:34,500 INFO    MainThread:42917 [backend.py:ensure_launched():205] started backend process with pid: 42946
2021-12-02 23:33:34,507 INFO    MainThread:42917 [wandb_init.py:init():458] backend started and connected
2021-12-02 23:33:34,508 INFO    MainThread:42917 [wandb_init.py:init():519] updated telemetry
2021-12-02 23:33:34,523 INFO    MainThread:42917 [wandb_init.py:init():549] communicating current version
2021-12-02 23:33:35,573 INFO    MainThread:42917 [wandb_init.py:init():554] got version response 
2021-12-02 23:33:35,573 INFO    MainThread:42917 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-02 23:33:35,833 INFO    MainThread:42917 [wandb_init.py:init():592] starting run threads in backend
2021-12-02 23:33:39,136 INFO    MainThread:42917 [wandb_run.py:_console_start():1816] atexit reg
2021-12-02 23:33:39,137 INFO    MainThread:42917 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-12-02 23:33:39,137 INFO    MainThread:42917 [wandb_run.py:_redirect():1695] Redirecting console.
2021-12-02 23:33:39,139 INFO    MainThread:42917 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-02 23:33:39,139 INFO    MainThread:42917 [wandb_init.py:init():619] run started, returning control to user process
2021-12-02 23:33:39,140 INFO    MainThread:42917 [wandb_watch.py:watch():43] Watching
2021-12-02 23:33:39,141 INFO    MainThread:42917 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-02 23:33:39,141 INFO    MainThread:42917 [wandb_run.py:_restore():1758] restore
2021-12-02 23:33:41,157 INFO    MainThread:42917 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 804
}

2021-12-02 23:33:41,389 INFO    MainThread:42917 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 804
}

2021-12-02 23:33:42,275 INFO    MainThread:42917 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 5905
}

2021-12-02 23:33:42,381 INFO    MainThread:42917 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 5905
}

2021-12-02 23:33:42,487 INFO    MainThread:42917 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 5905
}

2021-12-02 23:33:42,592 INFO    MainThread:42917 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5905
  total_bytes: 5905
}

2021-12-02 23:33:42,697 INFO    MainThread:42917 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5905
  total_bytes: 5905
}

2021-12-02 23:33:42,801 INFO    MainThread:42917 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5905
  total_bytes: 5905
}

2021-12-02 23:33:42,908 INFO    MainThread:42917 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5905
  total_bytes: 5905
}

2021-12-02 23:33:43,014 INFO    MainThread:42917 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5905
  total_bytes: 5905
}

2021-12-02 23:33:43,119 INFO    MainThread:42917 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5905
  total_bytes: 5905
}

2021-12-02 23:33:43,501 INFO    MainThread:42917 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5905
  total_bytes: 5905
}

2021-12-02 23:33:44,002 INFO    MainThread:42917 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5905
  total_bytes: 5905
}
local_info {
}

2021-12-02 23:33:45,309 INFO    MainThread:42917 [wandb_run.py:_append_files():2184] logging synced files
